xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -287,6 +287,10 @@ bool CGUIWindowMusicNav::GetDirectory(const std::string &strDirectory, CFileItem
       OnRetrieveMusicInfo(items);
   }
 
+  //Navigating music files so default content to "songs" unless in sources folder.
+  //This content allows view type to include media info so that file tag data can be displayed
+  if (!URIUtils::IsSourcesPath(strDirectory))
```

Comment:
This is wrong and is causing issues for root listings like the library node. You can't assume and set content `songs `for listings as it won't properly apply (eg. for folders). The song view types should be library exclusively and files node should have content `files` only. This is how it's handled in the video section.